modules/claims_api/app/controllers/claims_api/v2/base.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'claims_api/form_schemas'
+
+module ClaimsApi
+  module V2
+    class Base < ClaimsApi::V2::ApplicationController
+      private
```

Comment:
should these be `protected` since they're being accessed in classes that extend this class?

Reply:
Eh, I was just following the same pattern used in [v1](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/app/controllers/claims_api/v1/forms/base.rb#L23).

Reply:
fair enough, i think protected is "more valid" maybe? but not the end of the world as long as it functions 👍🏻